config/schema/format_strawberryfield.schema.yml

```diff
@@ -275,15 +297,15 @@ field.formatter.settings.strawberry_mirador_formatter:
       type: mapping
       label: 'Sources for IIIF URL'
       mapping:
-        manifestnodelist:
-          type: string
-          label: 'If manifestnodelist is being used'
-        metadataexposeentity:
-          type: string
-          label: 'If metadataexposeentity is being used'
-        manifesturl:
-          type: string
-          label: 'If manifesturl is being used'
+      manifestnodelist:
```

Comment:
Is this right? Or is this what is nor working for you? This used to be a sequence and i think i will revert to that

Reply:
@DiegoPino oof, I think it was the YAML indentation.

Reply:
@DiegoPino no need to revert It is fixed with mapping.